modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -57,3 +70,12 @@
 			if(pixel_x >= -16 + base_pixel_x)
 				pixel_x--
 				is_shifted = TRUE
+	if(pixel_x > 8 || pixel_x < -8)
+		is_out_of_the_way = TRUE
+	else if(pixel_y > 8 || pixel_y < -8)
+		is_out_of_the_way = TRUE
+	else
+		is_out_of_the_way = FALSE
+
+	if(is_out_of_the_way != was_out_of_the_way)
+		density = was_out_of_the_way
```

Comment:
That looks a little weird in terms of logic, what's the goal of this?
Why don't you just do this?
```suggestion
	density = is_out_of_the_way
```

Also, I can see this potentially causing issue when you pixel-shift while crawling, no?

Reply:
That's why the logic is so weird there. It's to make sure that density can't be set to true while crawling!

Reply:
Or otherwise having density set to true while it's set to false some other way-

Reply:
Wait, I screwed up the logic here...

Reply:
Uhhh, turns out setting density while prone seems to do nothing in my testing, so yay?

Reply:
Okay, good.